src/libServer/IsolatedServer.cpp

```diff
@@ -698,29 +738,26 @@ Json::Value IsolatedServer::CreateTransactionEth(EthFields const& fields,
 
       const Account* sender = AccountStore::GetInstance().GetAccount(fromAddr);
 
-      if (!ValidateTxn(tx, fromAddr, sender, m_gasPrice)) {
+      if (!ValidateEthTxn(tx, fromAddr, sender)) {
         return ret;
       }
 
+      senderBalance = uint256_t{sender->GetBalance()} * EVM_ZIL_SCALING_FACTOR;
```

Comment:
@moboware could you take a look if this makes sense and should be done like this?